resources/knative-eventing-kafka/values.yaml

```diff
@@ -16,11 +16,21 @@ image:
 
 # Networking Configuration for the Pods container and the K8S Service
 kafka:
+  #TODO Delete
+  namespace: namespaze
+  username: admin
+  password: pazzword
+  secretName: secretz
+  #TODO End Delete
   networking:
     container:
       metricsPort: 8081 # Port exposed by container for metrics
     service:
       metricsPort: 8081 # Port exposed by service for metrics
+  # Kafka Broker URL's (Comma Separated)
+  brokers:
+    hostname: namespace.servicebus.windows.net
```

Comment:
isn't it ***servicebus.windows.net**

Reply:
This is just a placeholder value, should be overriden by the charts values

Reply:
Right, can we add a comment to let the users know that, this is a dummy?

Reply:
OK